drizzlepac/hlautils/diagnostic_json_harvester.py

```diff
@@ -4,8 +4,12 @@
 drizzlepac/hlautils/svm_quality_analysis.py and stores it as a Pandas DataFrame"""
 
 # Standard library imports
+from collections import OrderedDict
 import glob
 import json
+import os
+import pdb
```

Comment:
This can be removed, since the debugging code that used it is no longer in the code.

Reply:
Issue resolved by commit [6f49d9f](https://github.com/spacetelescope/drizzlepac/pull/656/commits/6f49d9f71b248bb5fca4afc50b699b379b2a3855)